### PR TITLE
[ASCII-2115] add OTLP section in the GUI

### PR DESCRIPTION
**Pull request by GustavoCaso**
### What does this PR do?

Display OTLP status information in the GUI.

During the migration to use the status components  https://github.com/DataDog/datadog-agent/pull/22115 I missed the OTLP HTML section for the GUI

This PR fix that

### Describe how to test/QA your changes

Added unit tests and QA locally 
Screenshot 2024-08-01 at 15 44 46

### CODE REVIEW

**Review by pgimalac — APPROVED** [review]

**Comment by pgimalac** [comment]
Do you think there should be a bugfix releasenote ?

**Review by songy23 — COMMENTED** [review]
Please add changelog, I think this is a bug fix

**Review by songy23 — COMMENTED** [review]
Also is there a chance we can get this back-ported to 7.55 & 7.56? There are customers on 7.55 complaining about this

File: releasenotes/notes/fix-otlp-status-out-gui-7c34339ac01c1532.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+sfixes:
```
Comment (songy23) [inline releasenotes/notes/fix-otlp-status-out-gui-7c34339ac01c1532.yaml:9]: typo?

**Review by songy23 — APPROVED** [review]

File: releasenotes/notes/fix-otlp-status-out-gui-7c34339ac01c1532.yaml (hunk 1 above)
Comment (GustavoCaso) [reply releasenotes/notes/fix-otlp-status-out-gui-7c34339ac01c1532.yaml:9]: yes

**Review by cswatt — APPROVED** [review]
relnotes approved

**Pull request merged**